test/markup/latex/control_sequences.expect.txt

```diff
@@ -0,0 +1,34 @@
+<span class="hljs-keyword">\foo</span> bar
+<span class="hljs-keyword">\foo</span>:bar
+<span class="hljs-keyword">\ </span>and<span class="hljs-keyword">\
+</span>bar<span class="hljs-keyword">\foo</span>:baz
+<span class="hljs-keyword">\!</span>!
+<span class="hljs-keyword">\^^3a</span>bc
+<span class="hljs-keyword">\^^3</span>ABC
+<span class="hljs-keyword">\^^x</span>yz
+<span class="hljs-keyword">\^</span><span class="hljs-built_in">^</span>°!
```

Comment:
This is intentional?

Reply:
Yes. I wanted to do a negative test, too. (`°` does not qualify for the `^^` syntax, as it is not in ASCII.)